### DISCUSSION TITLE
[FEATURE REQUEST] Button to copy/duplicate a style preset

### DISCUSSION THREAD

**Discussion by iyukar** [Acly/krita-ai-diffusion#1084] [Ideas]
As many like to tinker with settings, and changing things back to how they were .. if you can remember, is not much fun.
The ability to quickly create a copy of a style preset with a hit of a button would be very convenient,
(it's possible to look for the right .json and copy the file .. but a button would be a more elegant and quick option)

**Comment by Acly** [comment]
Added in [v1.23.0](https://github.com/Acly/krita-ai-diffusion/releases/tag/v1.23.0)
